elife-00666.xml

```diff
@@ -1797,7 +1797,7 @@ not add all rids to the xref as it does not work. Only add the first one -->
                                     <title>Title of the source code.</title>
                                     <p>Legend of the source code.</p>
                                 </caption>
-                                <media mimetype="application" mime-subtype="xml" xlink:href="elife-00666-fig4-code1.xlsx"/>
+                                <media mimetype="application" mime-subtype="xlsx" xlink:href="elife-00666-fig4-code1.xlsx"/>
```

Comment:
Hi Chris

There is an entry on http://www.iana.org/assignments/media-types/media-types.xhtml for application and xml, so we're keeping it simple and just using the basic types for ease. Also, Graham does some jiggery pokery for the Crossref supply, so again need to keep simple in XML.